### PR TITLE
Feature/load initial state from storage

### PR DESCRIPTION
**Pull request by aschui**
You can now wrap a page with LoadStateWrapper which will read and save to localstorage and will show a loading page until client has loaded so the first load for client will show previous colours

### CODE REVIEW

**Comment by maxquinn** [comment]
You must be some kind of boy genius 🤔

File: pages/settings.jsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { useContext } from 'react';
+import React, { useContext } from 'react';
```
Comment (maxquinn) [inline pages/settings.jsx]: Don't need react in the pages. Next does that for us

Comment (maxquinn) [reply pages/settings.jsx]: Or the components for that matter

**Review by maxquinn — CHANGES_REQUESTED** [review]

File: pages/settings.jsx (hunk 1 above)
Comment (aschui) [reply pages/settings.jsx]: ahh, I didnt see this 

```
"rules": {
    "react/react-in-jsx-scope": "off"
  }
```

**Pull request merged**